README.md

```diff
@@ -154,10 +173,20 @@ taskViewController.delegate = self;
 [self presentViewController:taskViewController animated:YES completion:nil];
 ```
 
+Swift
+
+```swift
+let taskViewController = ORKTaskViewController(task: task, taskRunUUID: nil)
+taskViewController.delegate = self
+self.presentViewController(taskViewController, animated: true, completion: nil)
```

Comment:
To be consistent with our style, these should just get the “self” stripped off, so it’s
```
presentViewController(taskViewController, animated: true, completion: nil)